### PR TITLE
[Core][BlindText][Params] Clean out BlindText and remove CryptoPP Library and Upgrade ChainParams

### PR DESCRIPTION
**Pull request by empinel**
#### What is the purpose of this pull request (PR)?

To Clean out BlindText and remove CryptoPP Library and Upgrade ChainParams

#### Any background context to help the reviewer?

The BlindText feature is severely useless due to the existence of a messaging system

#### Was this PR tested and how (if not, why)?

Not yet, no - give me 24 hours

#### Does this PR resolve an open issue (reference issue #)?
1. Fixes issue # It Doesn't

#### Screenshots (if appropriate)

No

#### More Questions:
- Does the knowledge base need an update? Probably
- Does this PR add or change dependencies? Removes CryptoPP
- Does this change the chain or fork the network? it can't

### CODE REVIEW

**Comment by CryptoDJ** [comment]
The updates to the chain, the removal of CryptoPP and blindtext should be 3 pull request.  Closing this one.

**Comment by CryptoDJ** [comment]
For testing, we need the private key for the SandStorm (DarkSend) private key.  We will remove before release.  Also, I think Spencer wants to keep the public alert key in the code.

**Comment by empinel** [comment]
When did I remove that?

File: src/chainparams.cpp [hunk 1]
```diff
@@ -53,52 +87,61 @@ static void convertSeeds(std::vector<CAddress> &vSeedsOut, const unsigned int *d
...
+		/**
+		 ** The message start string is designed to be unlikely to occur in normal data.
+         ** The characters are rarely used upper ASCII, not valid as UTF-8, and produce
+         ** a large 4-byte int at any alignment.
+         **/
         pchMessageStart[0] = 0x1f;
         pchMessageStart[1] = 0x22;
         pchMessageStart[2] = 0x05;
         pchMessageStart[3] = 0x31;
-        vAlertPubKey = ParseHex("0450e0acc669231cfe2d0a8f0d164c341547487adff89f09e1e78a5299d204bd1c9f05897cb916365c56a31377d872abddb551a12d8d8163149abfc851be7f88ba");
```
Comment (CryptoDJ) [inline src/chainparams.cpp]: This removes the public alert key.

File: src/chainparams.cpp [hunk 2]
```diff
@@ -114,19 +157,14 @@ class CMainParams : public CChainParams {
 
         //TODO(AA)
         nPoolMaxTransactions = 3;
-        strSandstormPoolDummyAddress = "DDCxTmWLPytjnEAMd3TCGaryJStEx5caSm"; //private key = MpBeYuuA7c47bqa6ubmBnP8P7hkpmJTSUgwejC8AehSPwsXmkZHD
```
Comment (CryptoDJ) [inline src/chainparams.cpp]: This removes the private key for the SandStorm dummy address we are going to use for testing.

**Comment by CryptoDJ** [comment]
I made comments where you removed those two.  Both are in src/chainparams.cpp.

**Comment by empinel** [comment]
Look at line 119 and 160

**Pull request closed without merging**